### DISCUSSION TITLE
🆕  From the Actions World 🚢 - Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#61341] [Actions]
👋 Hey, everyone! Welcome to another exciting Actions-packed [community check-in!](https://github.com/orgs/community/discussions/categories/actions?discussions_q=is%3Aopen+category%3AActions+label%3A%22Community+Check-In%22) 
Get ready to explore the latest key changes in Actions, discover 🌐cool content from our Docs, and, of course, extend our heartfelt ❤️  thank you's to the amazing members of our community who make all of this possible. Let's dive in and celebrate the progress of their valuable contributions together!

## 💌  From the Community - Shoutout 📢  to these members

- A **BIG** round of applause 👏  for [ChristopherHX](https://github.com/ChristopherHX) for continuously delivering top solutions to the community. Let's check out some of their masterpieces:
    - [@pbrisbin](https://github.com/pbrisbin) had a hard time setting up required statuses using Conditional Jobs, [ChristopherHX](https://github.com/ChristopherHX) emulated and [diagnosed the issue](https://github.com/orgs/community/discussions/60792) with perfection. Well done!
    - Another one from our SuperStar ⭐ - corrected a missing output and [saved the day](https://github.com/orgs/community/discussions/60422) for [@NullVoxPopuli](https://github.com/NullVoxPopuli) Impressed by their clear documentation that aids others' learning too. Do check out their runner.server tool to emulate local GitHub Actions and Azure pipelines.
    - **Short and Sweet!** Kudos 👍  to [ChristopherHX](https://github.com/ChristopherHX) for sharing their expertise, guiding the community to [raise feature requests](https://github.com/orgs/community/discussions/60513) for functionalities. 👏. 
- Thank You 💟  [@jcubic](https://github.com/jcubic) for coming back to their [discussion](https://github.com/orgs/community/discussions/60925) related to Actions running unexpectedly slower. They found that their workflow was missing the Close() method causing Actions to wait longer. What an **incredible self-solve** 💯
- Let’s look at this self-solve ✔️ - **Much deserving** 👏applause for  [@Gitauwairimu](https://github.com/Gitauwairimu) for instantly [catching up with the erroneous workflow](https://github.com/orgs/community/discussions/60769) and sharing it with the community. 
- **Sparkles✨** for [@dumitrux](https://github.com/dumitrux) for doubling down on the issue at a glance and enabling [@flagrant99](https://github.com/flagrant99) to write [GitHub.json context to file](https://github.com/orgs/community/discussions/60515).

### [Removal of Node12 from the Actions runner](https://github.blog/changelog/2023-07-17-github-actions-removal-of-node12-from-the-actions-runner/) ℹ️ 

We started the [deprecation process of Node12](https://github.com/nodejs/Release/#end-of-life-releases) for GitHub Actions by migrating all actions to run on Node16 on June 14th. 

We have seen no major issues reported and a minimal number of people opted out of the forced upgrade to Node16. Given this, we will remove Node12 from the Actions runner on the 14th of August 2023.

**Future plans**: Check out [actions/runner nodejs plan and next steps](https://github.com/actions/runner/discussions/2704) 🤝 

**What do you need to do?** ❓ 

  - **For Actions maintainers**: Update your actions to run on Node16 instead of Node12 ([Actions configuration settings](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-javascript-actions))
  - **For Actions users**: Update your workflows with latest versions of the actions which runs on Node16 ([Using versions for Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions)) ✅ 

### [OIDC integration with AWS no longer requires pinning of intermediate TLS certificates](https://github.blog/changelog/2023-07-13-github-actions-oidc-integration-with-aws-no-longer-requires-pinning-of-intermediate-tls-certificates/)

GitHub Actions – OpenId Connect (OIDC) integration with AWS is now optimized to avoid pinning any intermediary certificate thumbprints. Learn 👀 more about [using OIDC with GitHub Actions](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect).

## Learning Actions? Great! but.. Did You Know? 💡

GitHub Actions has one of the most comprehensive documentation to help you learn and self-serve. But, did you know about **[Guides for GitHub Actions](https://docs.github.com/en/actions/guides)**? ❓ 

These guides include specific use cases and examples to help you configure workflows. It is packed with learning paths that are a collection of guides to help you master a particular CI/CD subject. We have **[54 guides](https://docs.github.com/en/actions/guides#all-guides)** available with amazing quickstart snippets, overview of each use-case, and tutorials to empower you to succeed with Actions. 

**Bonus: Discover the full potential of GitHub Actions** 🎉 

Learn from the seasoned industry experts - this **5+ hours training module** can help you unlock Actions full potential. Dive into 🏊  how to **[Automate your workflow with GitHub Actions](https://learn.microsoft.com/en-us/training/paths/automate-workflow-github-actions/)**.

## ☝️ GitHub Availability Report ➕

At GitHub, we put developers first and work hard to provide a safe, open, and inclusive platform for code collaboration. We are committed to be as transparent as we can and here to provide you updates on any disruption of services and with that in mind, we publish GitHub availability reports each month owning and explaining what happened and how we mitigated each specific incident.  GitHub [June availability report](https://github.blog/2023-07-12-github-availability-report-june-2023/) is now available. 

We update our [status page](https://www.githubstatus.com/) in real time in case of an ongoing incident about Actions. This has also been updated in the **[Actions Community FAQ](https://github.com/orgs/community/discussions/48283)**. If you see an open incident, this means that we are looking into it already. If not, please let us know about it in the [GitHub community](https://github.com/orgs/community/discussions/categories/actions).

➕ Till Next Time 🚀 

### Previously: [🎢 Rolling into July 🍹 - Actions Community Check-in 🤝](https://github.com/orgs/community/discussions/60108)

**Comment by GitLN01** [comment]
Yay, I'm so excited!
